transcom/mymove
# B 19730 int add state to us_post_region_cities

## [B-19730](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A945440)

## Summary
Adding a new column to show the state in the us_post_region_cities database table. The states are added by looking up the state from the re_zip3s database table using the first 3 digits of each uspr_zip_id (zip code) in the us_post_region_cities table.

Is there anything you would like reviewers to give additional scrutiny?

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Run the database migration
2. Verify the states are listed in the us_post_region_cities database table

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
